src/Controller/GarbageController.php

```diff
@@ -2,17 +2,21 @@
 
 namespace App\Controller;
 
+use App\Repository\GarbageRepository;
 use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
 use Symfony\Component\HttpFoundation\Response;
 use Symfony\Component\Routing\Annotation\Route;
 
 class GarbageController extends AbstractController
 {
     #[Route('/garbage', name: 'garbage')]
-    public function index(): Response
+    public function index(GarbageRepository $garbageRepository): Response
     {
+        $allGarbage = $garbageRepository->findById(1);
+        $totalGarbage = $allGarbage['nonRecycledWaste'] + $allGarbage['recycledWaste'];
```

Comment:
looks weird... if you find by id, your code should be:

    $garbage = $garbageRepository->find(1);
    $totalGarbage = $garbage->getNonRecycledWaste() + $garbage->getRecycledWaste();